src/main/java/org/opensearch/knn/jni/FaissService.java

```diff
@@ -145,6 +145,7 @@ public static native KNNQueryResult[] queryIndexWithFilter(
         long indexPointer,
         float[] queryVector,
         int k,
+        int queryEfSearch,
```

Comment:
nit :- Can we have all Places either Integer or int

Reply:
I will prefer being consistent here, This is a JNI layer and we seem to have been using primitive data types here, there is no real advantage of using `Integer` as we will later have to add lines to convert it to cpp int. We won't be able to avoid a dummy value in either case

I am not using `int` elsewhere because there is a possibility of value being absent which I feel is better than a dummy value.